### PR TITLE
fix issue events schema

### PR DESCRIPTION
**Pull request by daigotanaka**
# Description of change

Some of the properties were marked as "null" type.
Change them to nullable string type.

# Manual QA steps
 
Before the fix, the tap was throwing an exception with the null type issue.
After the fix, the tap correctly fetches the data and stdout.
 
# Risks
 - None since the scope is limited to the previously broken issue_events stream.
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @daigotanaka, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

**Comment by daigotanaka** [comment]
@cmerrick: Done signing the agreement.

**Comment by cmerrick** [comment]
You did it @daigotanaka!

Thank you for signing the Singer Contribution License Agreement.

**Comment by daigotanaka** [comment]
Just realized this could be a duplicated effort with #97 to solve #95, except that I corrected other fields on tops of description.

**Comment by luandy64** [comment]
@daigotanaka this looks good to me. I want to run some tests tomorrow, and then I can merge

File: tap_github/schemas/issue_events.json [hunk 1]
```diff
@@ -171,7 +171,8 @@
         },
         "milestone": {
           "type": [
-            "null"
+            "null",
+            "string"
```
Comment (luandy64) [inline tap_github/schemas/issue_events.json:175]: I think this is wrong.

I made an issue on this repo (#104) and the milestone there returned as an object:
```
'milestone': {'title': 'new-feature'}
```

Comment (luandy64) [reply tap_github/schemas/issue_events.json:175]: Oh, I was looking at the wrong milestone, but it's still an object:
```
           'milestone': {'closed_at': None,
                         'closed_issues': 0,
                         'created_at': '2021-02-05T15:49:29Z',
                         'creator': {'avatar_url': 'https://avatars.githubusercontent.com/u/19473996?v=4',
                                     'events_url': 'https://api.github.com/users/luandy64/events{/privacy}',
                                     'followers_url': 'https://api.github.com/users/luandy64/followers',
                                     'following_url': 'https://api.github.com/users/luandy64/following{/other_user}',
                                     'gists_url': 'https://api.github.com/users/luandy64/gists{/gist_id}',
                                     'gravatar_id': '',
                                     'html_url': 'https://github.com/luandy64',
                                     'id': 19473996,
                                     'login': 'luandy64',
                                     'node_id': 'MDQ6VXNlcjE5NDczOTk2',
                                     'organizations_url': 'https://api.github.com/users/luandy64/orgs',
                                     'received_events_url': 'https://api.github.com/users/luandy64/received_events',
                                     'repos_url': 'https://api.github.com/users/luandy64/repos',
                                     'site_admin': False,
                                     'starred_url': 'https://api.github.com/users/luandy64/starred{/owner}{/repo}',
                                     'subscriptions_url': 'https://api.github.com/users/luandy64/subscriptions',
                                     'type': 'User',
                                     'url': 'https://api.github.com/users/luandy64'},
                         'description': None,
                         'due_on': None,
                         'html_url': 'https://github.com/singer-io/tap-github/milestone/1',
                         'id': 6398418,
                         'labels_url': 'https://api.github.com/repos/singer-io/tap-github/milestones/1/labels',
                         'node_id': 'MDk6TWlsZXN0b25lNjM5ODQxOA==',
                         'number': 1,
                         'open_issues': 1,
                         'state': 'open',
                         'title': 'new-feature',
                         'updated_at': '2021-02-05T15:49:48Z',
                         'url': 'https://api.github.com/repos/singer-io/tap-github/milestones/1'},
```

Comment (daigotanaka) [reply tap_github/schemas/issue_events.json]: @luandy64 Thx for reviewing. My sample data didn't have milestones. I parsed your example and created the (sub) schema using https://pypi.org/project/getschema/ 

Your example contained due_on: None, so I manually changed it to `date-time`.

Comment (luandy64) [reply tap_github/schemas/issue_events.json]: Thank you, the bane of my existence is creating test data to validate these schemas

**Review by luandy64 — APPROVED** [review]

**Pull request merged**